spec/brcobranca/boleto/sicredi_spec.rb

```diff
@@ -145,4 +144,23 @@
       expect(File.exist?(tmp_file.path)).to be_falsey
     end
   end
+
+  it 'quando dígito verificador for 10 deve ser mapeado para 0', focus: true do
+    attributes = @valid_attributes.merge({
+      convenio: '2442725',
+      agencia: '0217',
+      conta_corrente: '42725',
+      byte_idt: 1,
+      posto: '24',
+      numero_documento: 25,
+      valor: 20.00,
+      data_documento: Date.parse('2015-01-18'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.